google/gax/_grpc_oauth2client.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2015, Google Inc.
```

Comment:
Cool. So this file goes away entirely once https://github.com/GoogleCloudPlatform/google-auth-library-python/pull/67 is merged, right?